ui/packages/catalog-ui-search/package.json

```diff
@@ -80,7 +80,7 @@
     "moment-timezone": "0.5.16",
     "mt-geo": "1.0.1",
     "openlayers": "4.6.5",
-    "plotly.js": "1.10.5",
```

Comment:
I remembered there being a reason for not updating plotly, and I'm not sure if it still applies: https://github.com/codice/ddf/pull/1610

> Downgrades plotly dependency to address security issues. Upgrading to address is not possible until they address this issue: plotly/plotly.js#895

So whoever tests this should ensure to try out the histogram and verify it works as expected in IE11.